rep-0144.rst

```diff
@@ -0,0 +1,139 @@
+REP: 144
+Title: ROS Package Naming
+Author: Vincent Rabaud <vincent.rabaud@gmail.com>
+Status: Draft
+Type: Informational
+Content-Type: text/x-rst
+Created: 28-Jan-2015
+Post-History:
+
+Abstract
+========
+
+This REP gives advice on how a ROS package should be named.
+It formalizes and extends conventions that were formerly described in [1]_.
+
+Motivation
+==========
+
+As the number of ROS packages increases, it is hard to quickly find a package
+and guess its functionality based on its name.
+Over time, the lack of naming conventions created problems like use of
+unexplained acronyms or packages with the same functionality but different names.
+
+FOr now, ROS package names translate directly to packages in supported Operating
+Systems: therefore, there is a flat global namerspace in which rules have to
+be followed.
+
+This REP proposes rules to name a ROS package properly.
+Some of those rules are mandatory, others merely advised.
+
+Package Naming
+==============
+
+Mandatory Rules
+---------------
+
+A package name is defined more strictly than with Debian at [2]_.
```

Comment:
We could mention here *why* we choose stricter rule: mainly since the package name is mapped to a namespace (which implies restrictions in most programming languages).